ABDataCollectionCore.js

```diff
@@ -1842,8 +1847,10 @@ module.exports = class ABDataCollectionCore extends ABMLClass {
 
          if (this.__throttleIncoming) clearTimeout(this.__throttleIncoming);
          this.__throttleIncoming = setTimeout(() => {
-            // using queuedParse() to responsively handle large datasets.
-            this.queuedParse(data.data || data, () => {
+            this.__dataCollection.load(async () => {
```

Comment:
One of the changes I was trying to make in the original creation of `queuedParse()` was to remove this `this.__dataCollection.load()` call.  We are no longer returning data from this fn() and instead the `queuedParse()` is actually using `this.__dataCollection.parse()` to load the data.  

Is there a reason we need to keep the `.load()` call?